.gitignore

```diff
@@ -26,9 +26,12 @@ pip-log.txt
 _build
 
 # Environment
-.env
+.env/
+.venv/
+env/
+venv/
 
 # IDE files
 .idea
 .vscode
-.DS_Store

```

Comment:
Any idea why this is being highlighted as changed?

Reply:
Ah, I figured it out. Somehow a stray `\r` character got in there, and I removed it.